plutus-ledger-api/src/Plutus/V1/Ledger/Api.hs

```diff
@@ -105,6 +105,12 @@ import qualified UntypedPlutusCore.Evaluation.Machine.Cek         as UPLC
 plutusScriptEnvelopeType :: Text
 plutusScriptEnvelopeType = "PlutusV1Script"
 
+plutusDatumEnvelopeType :: Text
```

Comment:
A couple of things I'm not sure about here:
- Should these be `PlutusV1*` types? Datums and redeemers should be the same across versions, and maybe even across languages.
- Should these be here? If they're not plutus-specific they should probably be defined in `cardano-api`. I mentioned this in the issue. But perhaps we should just put something here until they are...

Reply:
I'll ask in #ledger